main.go

```diff
@@ -70,16 +70,26 @@ func (hn *HackerNewsSource) Fetch(count int) (map[int]string, error) {
 	pages := count / 30
 	for i := 0; i <= pages; i++ {
 		resp, err := http.Get(HackerNewsURL + strconv.Itoa(pages))
-		handleError(err)
+		if err != nil {
```

Comment:
I added error checkin from http client to avoid panic when has issue with request